charts/modernization-api/templates/ingress.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
   annotations:
     cert-manager.io/cluster-issuer: "letsencrypt-production"
     nginx.ingress.kubernetes.io/service-upstream: "true"
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      more_set_headers "X-Frame-Options: Allow";
```

Comment:
question: Would you be able to try `X-Frame-Options: SAMEORGIN` so limit it to only executable from the domain?  

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options

We may need to look into `Content-Security-Policy` in the future.